README.md

```diff
@@ -2,12 +2,18 @@
 
 Adds an object to describe the criteria to qualify candidates to participate in a contracting process.
 
+If you are using the [Lots extension](https://extensions.open-contracting.org/en/extensions/lots/master/), [follow its guidance](https://extensions.open-contracting.org/en/extensions/lots/master/#usage) on whether to use `tender.lots` fields or `tender` fields.
+
 ## Legal context
 
-In the European Union, this extension's fields correspond to [eForms BG-702 (Selection Criteria)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+In the European Union, this extension's fields correspond to [eForms BG-702 (Selection Criteria) and BG-72 (Selection Criteria Second Stage Invite Number)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/).
 
 ## Examples
```

Comment:
I think these examples would benefit from fleshing out with realistic values, rather than just the descriptions of the fields, and with an introductory sentence like those in the [award criteria extension](https://extensions.open-contracting.org/en/extensions/awardCriteria/master/).

Reply:
agreed, I've attempted this in the new PR to the best of my understanding based on the eForms examples.